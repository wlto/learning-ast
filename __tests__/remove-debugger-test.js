const fs = require('fs');
const path = require('path');
const babel = require('@babel/core');
const plugin = require('../plugins/removeDebugger');

const example = fs.readFileSync(
  path.resolve(__dirname, '../src/remove-debugger.js'),
  'utf-8',
);

it('remove debugger statements', () => {
  const {code} = babel.transform(example, {plugins: [plugin]});
  expect(code).toMatchSnapshot();
});
