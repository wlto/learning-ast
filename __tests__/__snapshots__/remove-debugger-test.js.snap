// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`remove debugger statements 1`] = `
"function test() {
  const a = 5;
  const b = 6;
}"
`;
